__tests__/integration.test.ts

```diff
@@ -0,0 +1,47 @@
+import * as core from '@actions/core'
+import CloudWatch from 'aws-sdk/clients/cloudwatch'
+import {postBuildStatus} from '../src/cw-build-status'
+
+jest.mock('aws-sdk/clients/cloudwatch')
+jest.mock('@actions/core')
+
+const MOCK_NAMESPACE = 'GithubCI'
+const MOCK_REPO = 'ros-tooling/action-cloudwatch-metrics'
+const MOCK_RETURN = 'success'
+
+describe('Integration test suite', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+    jest.spyOn(core, 'getInput')
+        .mockReturnValueOnce(MOCK_NAMESPACE)  // namespace
+        .mockReturnValueOnce(MOCK_REPO)       // project-name
+        .mockReturnValueOnce(MOCK_RETURN)     // status
+  })
+
+  test('post build metrics successfully', async () => {
+    CloudWatch.prototype.putMetricData = jest.fn().mockImplementationOnce(() => {
+      return {
+        promise() {
+          return Promise.resolve({
+            error: null,
+            data: 'success'
```

Comment:
can you reference `MOCK_RETURN` here?

Reply:
It's not the same since this is the result of the data from the AWS API and not the input from the action.

Reply:
I said this going off both having the value `'success'`.
This is fine if they're actually not related.